docs/contribute/release.md

```diff
@@ -13,8 +13,8 @@ Releasing a new version of `plone.api` involves the following steps:
 To avoid human errors and to automate some of the tasks above we use `jarn.mkrelease`.
 It's listed as a dependency in `setup.py` and should already be installed in your local bin:
 
-```bash
-$ bin/mkrelease --help
+```shell
+bin/mkrelease --help
```

Comment:
I doubt this is correct.

Reply:
What would be correct?

Reply:
We use the `fullrelease` command from `zest.releaser`.
Really there is nothing special about releasing this package. I would say we completely remove the `release.md` file. It is outdated, seeming from a time when releasing packages to PyPI was new and needed to be explained.